.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 Please fill in the fields below to submit a pull request.  The more information
 that is provided, the better.
 
-**Fixes issue #**:
+**Fixes**:
```

Comment:
Maybe leave the '#' in there to suggest this is not just a title but something that can be filled in?
```suggestion
**Fixes #**
```

Reply:
... and maybe even add a placeholder?
Something like 
```suggestion
**Fixes #<ISSUE NUMBER>**
```